### ISSUE TITLE
Latest RN Version Support

### ISSUE THREAD

**Issue by privateOmega** [frostney/react-native-create-library#66]
Hi,
Is there any upcoming release, so as to support latest versions of RN and Gradle Plugins?

**Comment by maicki** [comment]
@privateOmega Hey! - Do you the latest master or the version on npm? I have a branch on my fork that updates RN on the library side (peerDependencies), but on master we also bootstrap an example. Which version do you think of?

**Comment by privateOmega** [comment]
@maicki Thanks. I will definitely look into your fork, but then again, Is there a scope for breaking? Also do you have any resource on building a library from scratch?

**Comment by maicki** [comment]
@privateOmega Just to clarify which version would you like to update? The version we are using as `peerDependendency` in a new library?

**Comment by wregis** [comment]
I guess it might be the `peerDependendency` version, since it points to `0.41.2` as for 5 minutes ago (I just installed the cli and create a test lib) while the current stable is 0.55.0.

I didn't really made any search for what were the breaking changes, however I think it would be good to have the version updated.

**Comment by maicki** [comment]
I thought about upgrading the `peerDependencies` and for now I have a branch out with updating it to `0.52`: https://github.com/maicki/react-native-create-library/commit/b90beb2f06773fffe3d29795e216278fd4b65dd8

The reason I'm not pretty sure if we should update always to the latest is that we know updating React Native is not so easy in some circumstances. So the `peerDependency` should rely define which version you should have at least as peer. If the library uses APIs that are only starting with `0.52+` you should manually update the `peerDependency` of the library. That said I think `0.52` should be a good next update step for a default library.

**Comment by wregis** [comment]
@maicki if the change is as simple as in your branch I guess it won't hurt to have it merged to the master. Anyway, I think it is nice to update it and do the necessary changes every once in a while.

**Comment by privateOmega** [comment]
@wregis I totally agree, cause the older dependency prevents developers from using updated features of RN, hence limiting them.
